### PR TITLE
Prestocheckoperator

### PR DESCRIPTION
**Pull request by artwr**
@mistercrunch This solves the problem of a single check. Now on to solving the problem for check between two sql queries and ratio or possibly interval check.

### CODE REVIEW

File: airflow/operators/presto_check_operator.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        self.hook = PrestoHook(presto_dbid=presto_dbid)
+        self.sql = sql
+
+    def run_check(self):
+        logging.info('Executing SQL check: ' + self.sql)
+        records = self.hook.get_records(hql=self.sql)
+        if not records:
+            return False
+        else:
+            if str(records[0][0]) in ('0', '',):
```
Comment (mistercrunch) [inline airflow/operators/presto_check_operator.py]: Let's look at all cells on the first row here.

Might also be good to have a PrestoHook.get_first method that would prevent from blowouts on the airflow side

**Comment by artwr** [comment]
@mistercrunch :)

File: airflow/operators/presto_check_operator.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        self.hook = PrestoHook(presto_dbid=presto_dbid)
+        self.sql = sql
+
+    def execute(self, execution_date=None):
+        logging.info('Executing SQL check: ' + self.sql)
+        records = self.hook.get_records(hql=self.sql)
+        if not records:
+            return False
+        else:
+            if any([ str(r) in ('0', '',) for r in records[0] ]):
```
Comment (mistercrunch) [inline airflow/operators/presto_check_operator.py]: return not any([bool(r) for r in records[0]])

**Comment by mistercrunch** [comment]
dbid are all conn_id now, rebase!

**Comment by artwr** [comment]
@mistercrunch Argh except that conn_id change was not into master for the Presto operator. I should have checked, and pushed nother commit instead of rebasing. Darn.

**Comment by mistercrunch** [comment]
It looks like you merged, not rebase.

For the record: `git pull --rebase origin master` will bring in the history and replay your commits on top.

Now it looks like your PR has all these commits in it ....

**Comment by mistercrunch** [comment]
Oh

**Comment by mistercrunch** [comment]
Snap, I clicked the wrong button, didn't mean to close, just comment.

**Comment by artwr** [comment]
@mistercrunch Closing this PR without merging

**Pull request closed without merging**